anagrafica/models.py

```diff
@@ -1730,10 +1730,10 @@ def ha_membro(self, persona, membro=None, figli=False, **kwargs):
 
     def __str__(self):
```

Comment:
Hm, questo metodo e' diventato un po' brutto. E' una stupidaggine, ma possiamo fare un po' di refactoring, nel modo in cui costruiamo la stringa? Inoltre, per favore usa `(inattiva)` invece di `[non attiva]`.

Reply:
La condizione con due test inline la possiamo fare diventare una property? Se abbiamo entrambi i lati in un inline if-not, non c'e' bisogno della negazione (`a if x else b` > `b if not x else b`). Poi per me va bene.